plugins/coverbrowser.koplugin/main.lua

```diff
@@ -226,152 +225,155 @@ function CoverBrowser:addToMainMenu(menu_items)
                         callback = function()
                             self:setupCollectionDisplayMode("list_image_filename")
                         end,
-                        separator = true,
                     },
                 },
-                separator = true,
             },
-            -- Misc settings
+        },
+    }
+
+    -- add Mosaic / Detailed list mode settings to File browser Settings submenu
+    -- next to Classic mode settings
+    if menu_items.filebrowser_settings == nil then return end
+    table.insert (menu_items.filebrowser_settings.sub_item_table, 4, {
+        text = _("Mosaic / Detailed list modes settings"),
```

Comment:
Should "Detailed" start with an uppercase (as it does in the display mode items), or be lowercase for this to read as a sentence ? @Frenzie ?